prism-agent/service/server/src/main/scala/io/iohk/atala/castor/controller/DIDRegistrarEndpoints.scala

```diff
@@ -16,14 +16,27 @@ import io.iohk.atala.iam.authentication.apikey.ApiKeyCredentials
 import io.iohk.atala.iam.authentication.apikey.ApiKeyEndpointSecurityLogic.apiKeyHeader
 import io.iohk.atala.iam.authentication.oidc.JwtCredentials
 import io.iohk.atala.iam.authentication.oidc.JwtSecurityLogic.jwtAuthHeader
+import sttp.apispec.Tag
 import sttp.model.StatusCode
 import sttp.tapir.*
 import sttp.tapir.json.zio.jsonBody
 
 object DIDRegistrarEndpoints {
 
+  private val tagName = "DID Registrar"
+  private val tagDescription =
```

Comment:
probably, it would be nice to fine a place for the following links in the description:
https://docs.atalaprism.io/docs/atala-prism/prism-cloud-agent/did-management
https://docs.atalaprism.io/tutorials/category/dids